### ISSUE TITLE
Couln'nt find case_insensitive option in High Level Client Term Query

### ISSUE THREAD

**Issue by Sangeetha-Murugesan** [elastic/elasticsearch-net#5407]
Hi,
I am using ElasticSearch version 7.11. My TermQuery only returns case-sensitive matching results,to avoid that I set case_insensitive as true in DSL term query like below and it returns data as expected.

```
{
    "query": {
        "term": {
            "title": {
                "value": "BIG THEORY",
                "case_insensitive": true
            }
        }
    }
}
```
But when I am trying to implement in code using High Level Client [Nest], could'nt find the option to set case_insensitive as true.

Is there any way to set it as true?

Thanks in Advance

**Comment by stevejgordon** [comment]
Hi @Sangeetha-Murugesan! Thanks for raising this. It's something I have in fact been working on that slightly dropped off my radar so this has been a useful reminder! I'm aiming to have it implemented in time for the next release. If it's blocking you then an option, albeit not extremely nice is to drop to the low-level client for this case and send your query as `PostData`. You can opt for a few different response types.

```c#
var postData = PostData.String(@"{
    ""query"": {
            ""term"": {
                ""name"": {
                    ""value"": ""STEVE"",
                    ""case_insensitive"": true
                }
            }
        }
    }");

var response = _client.LowLevel.DoRequest<SearchResponse<Dictionary<string, object>>>(HttpMethod.POST, "wss/_search", postData);
```

**Comment by Sangeetha-Murugesan** [comment]
@stevejgordon  Any option to implement it in High Level Client?

**Comment by stevejgordon** [comment]
@Sangeetha-Murugesan Yes, absolutely! I'm working on it as we speak, but it won't be available until the next release so I wanted to give you a workaround until we have a date for 7.12.0.

**Comment by Sangeetha-Murugesan** [comment]
@stevejgordon  Thanks !!

**Comment by stevejgordon** [comment]
PR is merged for 7.12.0 release.
